src/land/imsfv3_scf2ioda.py

```diff
@@ -92,27 +100,20 @@ def _read(self):
         qdflg = 0*sndv.astype('int32')
         errsc = 0.0*sncv
         errsd = 0.0*sndv
-        errsd[:] = 0.08
-        ncd.close()
-
-        times = np.empty_like(sncv, dtype=object)
+        errsd[:] = 40.
 
-        # get datetime from filename
-        str_date = re.search(r'\d{8}', self.filename).group()
-        my_date = datetime.strptime(str_date, "%Y%m%d")
-        start_datetime = my_date.strftime('%Y-%m-%d')
-        base_datetime = start_datetime + 'T18:00:00Z'
+        times = get_observation_time(self.filename, sncv, ncd)
 
-        for i in range(len(lats)):
-            times[i] = base_datetime
-            sndv[i] = 0.001*sndv[i]
+        ncd.close()
 
         # add metadata variables
-        self.outdata[('dateTime', 'MetaData')] = times
-        self.outdata[('latitude', 'MetaData')] = lats
-        self.outdata[('longitude', 'MetaData')] = lons
-        self.outdata[('height', 'MetaData')] = oros
-        self.varAttrs[('height', 'MetaData')]['units'] = 'm'
+        self.outdata[('dateTime', metaDataName)] = times
+        self.varAttrs[('dateTime', metaDataName)]['units'] = iso8601_string
+        self.varAttrs[('dateTime', metaDataName)]['_FillValue'] = long_missing_value
+        self.outdata[('latitude', metaDataName)] = lats
+        self.outdata[('longitude', metaDataName)] = lons
+        self.outdata[('height', metaDataName)] = oros
+        self.varAttrs[('height', metaDataName)]['units'] = 'm'
```

Comment:
```suggestion
        self.outdata[('stationElevation', metaDataName)] = oros
        self.varAttrs[('stationElevation', metaDataName)]['units'] = 'm'
```

@YoulongXia-NOAA and @CoryMartin-NOAA my toDo list for today is:
change `height`s to `stationElevation`
replace `-999` with `{float/int}_missing_value`
remove the replicate routine `imsfv3grid_scf2ioda.py`

@ClaraDraper-NOAA is back next week have her concur

Reply:
@BenjaminRuston, note errd use 40.0 mm (not 80.0 mm or 0.08 m before) and utc use 00z (not 18Z). This is what I checked @ClaraDraper-NOAA used in land DA workflow. For height to stationElevation, I am not sure as this data is not station alike data but a gridded data. For gridded data elevation, you should know what name will be used as a ioda converter organizer.

Reply:
@YoulongXia-NOAA yes the error is 40mm and the time is 00z in the changes I've made

the name to be used is `stationElevation` to keep this 2d field generic understand it is a virtual station
but maybe I can save that change to be with a batch of other land converter updates

Reply:
Great. Ben, thank you for hard work to make these changes. I still work on land-ims-proc with the reviewers and hopefully we can reach a final solution as soon as possible. Have a good weekend.